### ISSUE TITLE
Rules for "relative" indentations to be placed into "columns" of the default indentation

### ISSUE THREAD

**Issue by SampsonCrowley** [rubocop/rubocop#7945]
## Is your feature request related to a problem? Please describe.

I use atom as a text editor. Because of that, it always enforces code be put into "columns" based on the default indentation. Trying to put code inbetween a column is infuriating because Atom automatically deletes leading spaces in groups of your configured indentation. I very much enjoy that code is always indented at some multiple of my preferred indentation so I really don't try to fight this at all.

This causes problems with relative spacing rules.

## Describe the solution you'd like

I would like there to be options to enforce relative spacing to the next "column"

example:

```ruby
# notice the opening parenthesis here is "in-between" a "column" of two spaces
def method_with_even_letters(
                              first_param_here,
                              aligned_with_the_next_column,
                              after_the_paren_column_completes
                            )
  # and the closing parenthesis aligned with the previous "column" before param indent
  method_execution_code
end
```

Basically I want all new lines aligned with a multiple of 2 spaces

## Describe alternatives you've considered

disabling rubcops' parameter indentation rule

## Additional context

Easier to see with atoms' visuals:
![Screenshot from 2020-05-08 18-14-17](https://user-images.githubusercontent.com/22461638/81458476-cca88580-9157-11ea-84fd-0f75cac58db1.png)
